### ISSUE TITLE
Using PHATE with images

### ISSUE THREAD

**Issue by sunshineinsandiego** [KrishnaswamyLab/PHATE#177]
Hi - Thanks for creating this library. The original paper has an example use case with facial images from a video, but I couldn't find any mention of how the images were processed / analyzed or incorporated into the PHATE pipeline. Would it be possible to share how this was done?  Thanks!

**Comment by kmoon3** [comment]
In that specific example, we simply converted the images into vectors, similar to what is often done when analyzing MNIST. You can do that with any image as well. Alternatively, you could pass the images through some pretrained neural network (e.g. a CNN) and use some embedding layer as the input into PHATE.

**Comment by sunshineinsandiego** [comment]
Thanks Kevin, did you dimensionality reduce / normalize the embedding vectors before passing them to PHATE? What were the original embedding dimensions and the input dimensions to PHATE that you used / what was the phate config that you used for the images? I'm trying to use this API:

phate_operator = phate.PHATE(knn=5, decay=15, n_jobs=-2, verbose=True)
adata.obsm['X_phate'] = phate_operator.fit_transform(adata.X)

from your tutorial with an N*d matrix of embeddeding vectors , but not sure what the phate operator expects. Thanks!
